tests/Doctrine/Tests/Common/Proxy/MagicSetClassWithScalarTypeAndRenamedParameters.php

```diff
@@ -24,7 +24,7 @@ class MagicSetClassWithScalarTypeAndRenamedParameters
      *
      * @throws BadMethodCallException
      */
-    public function __set($n, $val)
```

Comment:
This class is part of a different test and it should keep `__set($n, $val)` to cover cases where no types are used. Your case is already covered with `MagicSetClassWithScalarTypeAndVoidReturnType`.

Reply:
I'll undo that change, but the `__set()` there should definitely have a scalar type in it's signature, otherwise the name of the test is misleading. As I understood it, the `MagicSetClass` class does not have any type in its signature, while the `MagicSetClassWithScalarType` class does have a type in its signature, hence the name. So the `MagicSetClassWithScalarTypeAndRenamedParameters` should either be renamed to `MagicSetClassWithRenamedParameters` or have the type added, to make this consistent. But that could also be proposed in yet another PR, as it does not impact the changes introduced with this one.

Reply:
I don't know about the intention of the naming, so let's keep those names for now, while keeping the original state of this `__set()`. Your new test will cover the type.

Reply:
Agreed. I'm going to revert this change now and hand in the PR for a re-review :)